diff --git a/dist/index.mjs b/dist/index.mjs
index 0ef28fc..602e734 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -3203,7 +3203,7 @@ const publicPathTemplate = {
 		return [
 			"import { joinRelativeURL } from 'ufo'",
 			!nuxt.options.dev && "import { useRuntimeConfig } from 'nitropack/runtime'",
-			nuxt.options.dev ? `const getAppConfig = () => (${JSON.stringify(nuxt.options.app)})` : "const getAppConfig = () => useRuntimeConfig().app",
+			nuxt.options.dev ? `const getAppConfig = () => (${JSON.stringify(nuxt.options.app)})` : "const getAppConfig = () => useRuntimeConfig()?.app ?? {}",
 			"export const baseURL = () => getAppConfig().baseURL",
 			"export const buildAssetsDir = () => getAppConfig().buildAssetsDir",
 			"export const buildAssetsURL = (...path) => joinRelativeURL(publicAssetsURL(), buildAssetsDir(), ...path)",
